public/docs/_examples/dependency-injection/dart/lib/app_component_1.dart

```diff
@@ -1,17 +1,17 @@
 // Early versions
 
 // #docregion
-import 'package:angular2/core.dart';
+import 'package:angular2/angular2.dart';
```

Comment:
I missed that recommendation. Can you tell me more about it? When talking to an internal engineer who uses Angular (but isn't on the Angular team), it seemed to us that angular2.dart was better because you wouldn't have to think twice about which files to include. (And tree shaking would make it not matter if you included more than you needed.)